### PR TITLE
feat(bridge): Show pause icon if sequence is paused

### LINKED ISSUE #6986: Show pause icon instead of sequence icon for paused sequences
When a sequence is paused, the black icon for the sequence is shown.
It would be more visible to the user that a sequence is paused, if we would show a pause icon instead next to the sequence name and stage name (timeline).

Environment screen:
image

## Acceptance Criteria

- [ ] Show the pause icon in the dashboard, environment, and sequence screen when the sequence state is paused

## DoD

- [ ] UI tests

### PR DESCRIPTION
**Pull request by TannerGilbert**
## This PR

- Adds pause icon that shows if a sequence is paused

### Related Issues

Resolves #6986

### Notes

dashboard-page

environment-page

sequences-page

### CODE REVIEW

File: bridge/client/app/_models/sequence.ts [hunk 1]
```diff
@@ -76,9 +76,12 @@ export function createSequenceStateInfo(sequence: ISequence, stageName?: string)
   const warning = !faulty && stages.some((s) => s.latestEvaluation?.result === ResultTypes.WARNING);
   const successful = finished && !faulty && !warning && !aborted;
   const pendingApproval = stages.some((s) => pendingApprovalTypes.includes(s.latestEvent?.type as EventTypes));
-  const icon = latestStage?.latestEvent?.type
-    ? EVENT_ICONS[getShortType(latestStage?.latestEvent?.type)] || EVENT_ICONS.default
-    : EVENT_ICONS.default;
+  const icon =
```
Comment (renepanzar) [inline bridge/client/app/_models/sequence.ts:79]: This statement is already hard to understand - please simplify it (extract into function, replace ternary operators with `if`s) and write unit tests for it.

File: bridge/client/app/_models/sequence.ts [hunk 2]
```diff
@@ -275,7 +278,9 @@ export class Sequence implements ISequence {
 
   public getIcon(stageName?: string): DtIconType {
     const stage = stageName ? this.getStage(stageName) : this.stages[this.stages.length - 1];
-    return stage?.latestEvent?.type
+    return this.isPaused()
```
Comment (renepanzar) [inline bridge/client/app/_models/sequence.ts:281]: Please also extract this statement into a function, so it can easily be tested and maintained.

**Review by renepanzar — CHANGES_REQUESTED** [review]
Please simplify according to the comments.

File: bridge/cypress/fixtures/sequences.sockshop.json [hunk 3]
```diff
@@ -480,29 +498,6 @@
...
-          },
-          "latestEvent": {
-            "type": "sh.keptn.event.staging.rollback.finished",
-            "id": "5d16ec19-e0ef-453d-b312-03e5988a0905",
-            "time": "2021-07-15T12:13:40.942Z"
-          },
-          "latestFailedEvent": {
-            "type": "sh.keptn.event.staging.rollback.finished",
-            "id": "5d16ec19-e0ef-453d-b312-03e5988a0905",
-            "time": "2021-07-15T12:13:40.942Z"
```
Comment (Kirdock) [inline bridge/cypress/fixtures/sequences.sockshop.json:505]: What is the reason why you removed the staging stage and latestEvent of the dev stage here?

File: bridge/cypress/integration/sequences.spec.ts [hunk 4]
```diff
@@ -371,5 +371,12 @@ describe('Sequences', () => {
 
       sequencePage.assertFilterItemsCount('Service', 5).clickFilterViewMore('Service').assertFilterShowMoreCount(12);
     });
+
+    it.only('should show pause icon if sequence is paused', () => {
```
Comment (Kirdock) [inline bridge/cypress/integration/sequences.spec.ts:375]: Could you remove `only`?

File: bridge/cypress/integration/sequences.spec.ts [hunk 5]
```diff
@@ -371,5 +371,12 @@ describe('Sequences', () => {
...
+      sequencePage.visit('sockshop');
+      cy.byTestId('keptn-root-events-list-e28592c6-d857-44fe-aea6-e65de02929bf')
+        .find('path')
+        .should('have.attr', 'd', 'M112 64h104v384H112zM296 64h104v384H296z');
```
Comment (Kirdock) [inline bridge/cypress/integration/sequences.spec.ts:379]: I guess there isn't an easier way to check this right? I don't know if `ng-reflect-name` would be the better choice here.
Even though, could you move this to `commands.ts` of Cypress and refactor the usages to use the same command? Something like `.assertDtIcon(type: ...)` with a switch case (map type to path attribute)

File: bridge/client/app/_models/sequence.ts [hunk 6]
```diff
@@ -50,6 +50,14 @@ function getStatusText(
   return status;
 }
 
+function getIcon(state: SequenceState | undefined, stage: SequenceStage): DtIconType {
+  if (state == SequenceState.PAUSED) return EVENT_ICONS.pause;
```
Comment (Kirdock) [inline bridge/client/app/_models/sequence.ts:54]:
```suggestion
  if (state === SequenceState.PAUSED) return EVENT_ICONS.pause;
```

**Review by Kirdock — CHANGES_REQUESTED** [review]

File: bridge/client/app/_models/sequence.ts (hunk 1 above)
Comment (heinzburgstaller) [reply bridge/client/app/_models/sequence.ts]: There are also unit tests for this function. Please extend them accordingly

**Review by heinzburgstaller — CHANGES_REQUESTED** [review]

**Review by Kirdock — APPROVED** [review]

**Review by heinzburgstaller — APPROVED** [review]

**Review by renepanzar — APPROVED** [review]
LGTM now 👍

**Pull request merged**